child_compassion/models/field_office.py

```diff
@@ -113,3 +117,62 @@ class FieldOfficeHighRisks(models.Model):
     res_field = 'high_risk_ids'
 
     value = fields.Char(translate=True)
+
+
+class FieldOfficeDisasterAlert(models.Model):
+    _name = 'fo.disaster.alert'
+    _description = 'FO disaster alert'
+    res_model = 'compassion.field.office'

```

Comment:
`res_model` and `res_field` are not needed. Those are only used in models that inherits from `connect.multipicklist`